src/extension.ts

```diff
@@ -189,27 +261,39 @@ export function activate(context: vscode.ExtensionContext) {
     } else {
       clearConnection();
     }
+
+    updateConnectionStatusBarItemVisibility();
   }
 }
 
 export function deactivate() {}
 
-async function ensureUriEditorIsActive(uri: vscode.Uri) {
-  const isActive =
-    uri.toString() === vscode.window.activeTextEditor?.document.uri.toString();
+async function ensureUriEditorIsActive(
```

Comment:
Better function might be `getEditorForUri` or `openEditorForUri` now that it's returning the editor... with a note that it makes it the active document.